jmmorrone/master-recipe
# Added auth for all endpoints

## What did you implement:
* Added Authentication for every endpoint. To access each endpoint, you need to have a Bearer token.

## How did you implement it:
* Configurated and created endpoints that communicate with Auth0 API to generate a Bearer token ready to use until its expiration time

## How can we verify it:
* Access /login and get your Bearer code.
* If you try to access any other endpoint with the Bearer token, you should see a valid response. If not, you'll see a 401 status.
